### DISCUSSION TITLE
macOS desktop app

### DISCUSSION THREAD

**Discussion by andrew6767** [osmandapp/OsmAnd#19307] [General]
Are there any plans for a desktop version osmAnd for macOS? should be fairly simple to port the iOS app over these days. Would love to have a version of osmAnd for the big screen to use in conjunction with the mobile app.

**Comment by DmitryAlexei** [comment]
Have you tried web version https://osmand.net/map ? You can synchronize it with your OsmAnd app via OsmAnd Cloud

**Comment by dmpr0** [comment]
OsmAnd Maps is available in [Mac App Store](https://apps.apple.com/app/osmand-maps-travel-navigate/id934850257) for new Mac's based on Apple silicon.

**Comment by trymeouteh** [comment]
Would like to see a desktop app for Windows, MacOS and Linux

**Reply by DmitryAlexei** [reply]
OsmAnd is available on Macs with arm chips. And probably OsmAnd will also work on windows PCs with arm chips.

**Reply by realgooseman** [reply]
> OsmAnd is available on Macs with arm chips. And probably OsmAnd will also work on windows PCs with arm chips.

What about x86/x64 Windows or Linux ? Will there ever be a native port ?
Web version is good but doesn't support map downloading and offline usage as far as I can tell.

**Comment by mariush444** [comment]
https://github.com/osmandapp/OsmAnd/discussions/20160#discussioncomment-11571648
